yarn.lock

```diff
@@ -11481,9 +11481,8 @@ inherits@2.0.3:
   resolved "https://registry.yarnpkg.com/inherits/-/inherits-2.0.3.tgz#633c2c83e3da42a502f52466022480f4208261de"
 
 ini@^1.3.4, ini@^1.3.5, ini@~1.3.0:
-  version "1.3.7"
-  resolved "https://registry.yarnpkg.com/ini/-/ini-1.3.7.tgz#a09363e1911972ea16d7a8851005d84cf09a9a84"
-  integrity sha512-iKpRpXP+CrP2jyrxvg1kMUpXDyRUFDWurxbnVT1vQPx+Wz9uCYsMIqYuSBLV+PAaZG/d7kRLKRFc9oDMsH+mFQ==
+  version "1.3.5"
+  resolved "https://registry.yarnpkg.com/ini/-/ini-1.3.5.tgz#eee25f56db1c9ec6085e0c22778083f596abf927"
```

Comment:
Any idea where this came from? I think we should revert it - 1.3.5 contains a security vulnerability and was updated to 1.3.7 via dependabot in https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1878

Reply:
I think thats a mistake. I originally updated carbon charts and then reverted and it looks like that slipped in. I'll swap it back

Reply:
Fixed!